python/holidays/hr-HR.py

```diff
@@ -9,12 +9,13 @@ class hr_HR(Locale):
     01-01: [NF] Nova Godina
     01-06: [NRF] Bogojavljanje / Sveta tri kralja
     05-01: [NF] Praznik rada
+    05-30: [NF] Dan državnosti   
     06-22: [NF] Dan antifašističke borbe
-    06-25: [NF] Dan državnosti
     08-05: [NF] Dan domovinske zahvalnosti
     08-15: [NRF] Velika Gospa
     10-08: [NF] Dan neovisnosti
     11-01: [NRF] Dan svih svetih
+    11-18: [NF] Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje
```

Comment:
Is this a new holiday for 2020 or did it exist before? Is this the official name (it seems unusual long...)?

Reply:
> Is this a new holiday for 2020 or did it exist before? Is this the official name (it seems unusual long...)?
Yes, this is a new on valid from 2020 onwards and such is the name - you can check the above-mentioned official document.

Sincerely,
Gour